specification/datafactory/resource-manager/Microsoft.DataFactory/preview/2017-09-01-preview/entityTypes/Trigger.json

```diff
@@ -324,6 +324,13 @@
             "retryPolicy": {
               "$ref": "#/definitions/RetryPolicy",
               "description": "Retry policy that will be applied for failed pipeline runs."
+            },
+            "dependsOn": {
+              "type": "array",
+              "description": "Tumbling Window depends on condition.",

```

Comment:
In your case there is no condition, so suggested description: "Tumbling window triggers that this trigger depends on."

Reply:
Resolved